plugin.json

```diff
@@ -22,6 +22,12 @@
         "type": "bool",
         "help_text": "If set the plugin will reject posts containing profanity instead of censoring."
       },
+      {
+        "key": "ExcludeBots",
+        "display_name": "Exclude Bots",
+        "type": "bool",
+        "help_text": "If set the plugin will exclude bot messages from being checked."
+      },
```

Comment:
Could you please move this setting before `Reject Posts`? That way `Reject Posts` and `Warning Message` stay next to each other because they are related.

Reply:
Done :)